tests/src/Functional/CommerceBrowserTestBase.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Drupal\Tests\commerce\Functional;
+
+use Drupal\Component\Render\FormattableMarkup;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Base class for Drupal Commerce (and contrib) to test with BrowserTest.
```

Comment:
"Provides a base class for Commerce functional tests."
(Drupal Commerce is just Commerce throughout the code, the contrib part is assumed)